### PR TITLE
Implement "add FIDO2 device"

### PR DESCRIPTION
**Pull request by roxanemace**
**Asana ticket**: https://app.asana.com/0/1111569008372652/1181397055064768

### CODE REVIEW

File: sdk-facebook/src/main/java/com/reach5/identity/sdk/facebook/FacebookProvider.kt [hunk 1]
```diff
@@ -93,7 +94,12 @@ class ConfiguredFacebookProvider(
 
             override fun onError(error: FacebookException?) {
                 if (error != null) {
-                    failure(ReachFiveError.from(error))
+                    if (error is FacebookAuthorizationException) {
```
Comment (rchapel) [inline sdk-facebook/src/main/java/com/reach5/identity/sdk/facebook/FacebookProvider.kt:97]: Is this change related to webauthn ?

File: app/build.gradle [hunk 2]
```diff
@@ -17,23 +17,29 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
-    buildTypes {
+    signingConfigs {
         release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro"
-            signingConfig signingConfigs.debug
+            storeFile file("keystore.jks")
```
Comment (rchapel) [inline app/build.gradle:22]: What's in this file ? And how it is generated ?

File: sdk-facebook/src/main/java/com/reach5/identity/sdk/facebook/FacebookProvider.kt (hunk 1 above)
Comment (roxanemace) [reply sdk-facebook/src/main/java/com/reach5/identity/sdk/facebook/FacebookProvider.kt:97]: Nope, I had an issue with my Facebook session and I couldn't logout. With this change, if there is a Facebook error, the user is automatically logout.

File: app/build.gradle (hunk 2 above)
Comment (roxanemace) [reply app/build.gradle:22]: This is a security certificate of the application required by FIDO2. It will be documented when I'll be done with the development.

Comment (rchapel) [reply app/build.gradle:22]: The `keystore` contains the private key to sign the application. Does it contains sensitive information ? Because the repository is public.

Comment (roxanemace) [reply app/build.gradle:22]: Nope, I don't see any sensitive information stored in it (do you have an idea of data?)

Comment (rchapel) [reply app/build.gradle:22]: Did you generated this file by yourself ? (In this case is will be ok to commit it).

Comment (roxanemace) [reply app/build.gradle:22]: Yes, I am the one who generated the file.
